suite/tests/client-interface/inline.dll.c

```diff
@@ -207,7 +207,7 @@ codegen_instrumentation_funcs(void)
         dr_log(dc, LOG_EMIT, 3, "Generated instrumentation function %s at "PFX
                ":", func_names[i], pc);
         instrlist_disassemble(dc, pc, ilists[i], dr_get_logfile(dc));
-        pc = instrlist_encode(dc, ilists[i], pc, false);
+        pc = instrlist_encode(dc, ilists[i], pc, true);
```

Comment:
Also interesting: this test sets its own note fields, which seems a little hacky.  Please update the comment up above on line 201-201 (I can't seem to add a review comment there -- another limitation of github reviews...) b/c we are now telling the encoder to "worry about labels" and it will clobber the notes -- so should also remove the set_note at line 189.